### ISSUE TITLE
Add colours dynamically.

### ISSUE THREAD

**Issue by geftimov** [geftimov/android-patternview#15]

**Comment by geftimov** [comment]
The enhancement was mentioned in #8.

**Comment by geftimov** [comment]
The method names are :

``` java
patternView.setPathColor(pathColor);
```

``` java
patternView.setCircleColor(pathColor);
```

``` java
patternView.setDotColor(pathColor);
```

**Comment by geftimov** [comment]
Fix at version [1.0.6](https://github.com/geftimov/android-patternview/releases/tag/1.0.6).
